database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
Observations: The code does not handle database connection errors gracefully, which can lead to a poor user experience.

Recommendations:
- **Graceful Error Handling**: Implement a mechanism to handle database connection errors and display a user-friendly message.
- **Example**:
  ```php
  if (!$connection) {
      die("<p>Failed to connect to database. Please try again later.</p>");
  }
  ```